Files/Views/MainPage.xaml

```diff
@@ -327,7 +328,6 @@
                                 Height="32"
                                 Padding="8"
                                 HorizontalAlignment="Right"
-                                x:Load="{x:Bind AppSettings.IsVerticalTabFlyoutEnabled, Mode=OneWay}"
```

Comment:
Is this correct? We have an option in settings to show the vertical tab flyout button or not